saltychat/SaltyClient/Model.cs

```diff
@@ -492,6 +492,7 @@ public RadioCommunication(string name, RadioType senderRadioType, RadioType ownR
             this.Direct = direct;
             this.Secondary = isSecondary;
             this.RelayedBy = relayedBy;
+            this.Volume = volume;
```

Comment:
Otherwise conditional property serialization wouldn't make sense for the `Volume` prop.
```suggestion

            if (volume != 1f)
                this.Volume = volume;
```